examples/air-quality-sensor-app/cc32xx/main/Logging.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
```

Comment:
Is this really copyright 2020?  In general, please check the copyright notices in all these files.